ui/utils/utils.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * captitalizestring capitalises the string presented to it
+ * "hello world!" -> "Hello World!"
+ * "hello_world!" -> "Hello World"
+ *
+ * @param {string} str The uncapitalised string
+ *
+ * @returns {string} the capitalised string
+ */
+export function capitalizeString(str) {
+  return str.split(/[\s|_]/) // separating regex for "_" or " "
+    .map(smallStr => smallStr.charAt(0).toUpperCase() + smallStr.substring(1).toLowerCase()) // transforming string
+    .join(" ") // join by whitespace
+}
```

Comment:
Same as previous comment, can we change the file name a little bit?

Reply:
Something like **StringUtils.js** would be better I guess, so that in future if any string manip functions need to be defined, everything can be done in this file itself

Reply:
Thanks a lot @bariqhibat @adithyaakrishna. Yeah that makes sense. Sorry I was missing the notifications here :(